src/main/kotlin/com/terraformation/backend/search/field/UriField.kt

```diff
@@ -25,13 +23,13 @@ class UriField(
     get() = EnumSet.of(SearchFilterType.Exact, SearchFilterType.PhraseMatch)
 
   override fun getCondition(fieldNode: FieldNode): Condition {
-    val valuesRegex = fieldNode.values.map { "$it" }
+    val nonNullValues = fieldNode.values.filterNotNull()
     return when (fieldNode.type) {
       SearchFilterType.PhraseMatch,
       SearchFilterType.Exact ->
           DSL.or(
               listOfNotNull(if (fieldNode.values.any { it == null }) databaseField.isNull else null)
-                  .plus(valuesRegex.map { databaseField.likeRegex(it) }))
+                  .plus(nonNullValues.map { databaseField.likeRegex(it) }))
```

Comment:
Not sure `likeRegex` is the operation we want here. The search API doesn't expect regexes as search strings in other places; it treats the search terms as literal strings even if they include `.` or `*` or other regex special characters.

Is the client expecting to be able to pass in actual regexes for searches on URI fields?